src/Pim/Component/Catalog/Factory/ProductValue/OptionsProductValueFactory.php

```diff
@@ -124,6 +132,16 @@ protected function getOptions(AttributeInterface $attribute, array $data)
             }
         }
 
+        if (empty($options) && !empty($data)) {
+            throw InvalidOptionException::validEntityCodeExpected(
```

Comment:
it's not an option
can't we build a custom exception?

same comment goes everywhere